test/parallel/test-http2-compat-serverrequest-headers.js

```diff
@@ -23,7 +23,6 @@ server.listen(0, common.mustCall(function() {
 
     assert.strictEqual(request.method, expected[':method']);
     assert.strictEqual(request.scheme, expected[':scheme']);
-    assert.strictEqual(request.path, expected[':path']);
```

Comment:
I think we should add an assertion that `request.path`  is `undefined` for express compatibility.

Reply:
Good thinking, will update.